Jarvis-Platform/jarvis-studio
# Feature/item component not found status

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Add not-found status on `ItemComponent` to avoid errors.


## Todos
- [x] Add `isNotFound` property on `src/mixins/data-workflows/item-mixin`
- [x] Add condition in `ItemComponent` template


## Deploy Notes
Review & merge


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout item-component-not-found-status
```

1. Load a listing page
2. Click on an Item to show its `ItemView`
3. Add letters to `:id` parameter in the URL
4. Load the page, you should the the "Document not found" message


## Impacted Areas in Application
All views using `ItemComponent`
